infrastructure/ansible/roles/traffic_ops/defaults/main.yml

```diff
@@ -101,6 +102,10 @@ to_go_read_header_timeout: 120
 to_go_read_timeout: 120
 to_go_write_timeout: 120
 
+to_smtp_enabled: false
+to_smtp_username: ""
```

Comment:
Unless blank is meaningful, a default should exist.

Reply:
username and password can be blank for smtp as some smtp servers dont require it to authenticate I would rather leave these empty specially given the default for smtp_enabled is false.